pg_lakehouse/README.md

```diff
@@ -0,0 +1,221 @@
+<h1 align="center">
+  <img src="../docs/logo/pg_lakehouse.svg" alt="pg_lakehouse" width="500px">
+<br>
+</h1>
+
+## Overview
+
+`pg_lakehouse` is an extension that transforms Postgres into an analytical query engine over data lakes like S3. Queries are pushed down to [Apache DataFusion](https://github.com/apache/datafusion), which significantly improves performance. Any combination of the following object stores, table formats, and file formats is supported.
+
+### Object Stores
+
+- [x] Amazon S3
+- [x] Local file system
+- [ ] Azure Blob Storage (coming soon)
+- [ ] Google Cloud Storage (coming soon)
```

Comment:
Have you considered using native extensions that come from cloud providers? 

for s3, there is aws_s3 - https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_PostgreSQL.S3Import.html#USER_PostgreSQL.S3Import.FileFormats

for azsure blob - seems like there is azure_storage - https://learn.microsoft.com/en-us/azure/postgresql/flexible-server/concepts-storage-extension#import-data-from-azure-blob-storage-to-azure-database-for-postgresql-flexible-server